### DISCUSSION TITLE
About admission_location variable

### DISCUSSION THREAD

**Discussion by EstebanGarciaG** [MIT-LCP/mimic-code#1365] [MIMIC-General] [answered]
Hello,

I want to ask about the location admission_location that gives information about the patient location prior to arriving at the hospital, specifically I am interested in the 'TRANSFER FROM HOSPITAL' patients, I would like to know what exactly it means and how the transfers to Beth Israel work, in order to have a better basis for making assumptions about whether or not I should exclude said patients from my cohort of interest.

I really appreciate everyone's help.

Best regards.

**Comment by briangow** [comment]
@EstebanGarciaG , I don't know the details behind 'TRANSFER FROM HOSPITAL' but can ask around to see if I can figure it out. It isn't clear what you need to know though. Can you let me know your criteria for including / excluding patients based on this and I'll try to get information around those specific criteria.

**Reply by EstebanGarciaG** [reply]
Thank you very much @briangow , 

We are interested in patients with community-acquired pneumonia, so we are excluding patients who acquired the pneumonia in the hospital (using diagnoses such as ventilator-associated pneumonia or checking the date/time of administration of antibiotics), however, with the 'HOSPITAL TRANSFER' patients we are not sure if the patients developed the pneumonia in the previous hospital and are being referred for treatment, as that is a large number of patients in relation to our cohort, we would like to know more about what it means so that we can make a better decision in terms of letting them into the cohort or not.

I really appreciate your attention and I look forward to your response.

Best regards.

**Comment by briangow** [comment, marked as answer]
@EstebanGarciaG , the only way I'm aware of to get the information you are looking for is to search the notes (currently available in MIMIC-III but not yet available in MIMIC-IV). For `TRANSFER FROM HOSP/EXTRAM` patients you will often find information under the History of Present Illness section in the notes which indicates what condition they presented with at the previous hospital.

**Reply by EstebanGarciaG** [reply]
Thanks a lot @briangow, 

I see, I understand the notes will be avilable in MIMIC-IV at some point but there is not a date rigth now. 

I will be attentive for that.

Best regards.

**Comment by Lauramoffat** [comment]
My guess is it was a transfer from a medical unit an ICU unit within BIDMC or a transfer from one of Beth Israel Lahey Health System’s other smaller hospitals to BIDMC
Laura


Laura M Moffat, DNP, MSK, APRN, AGCNS-BC, CMSRN
Clinical Assistant Professor
School of Nursing

Hampton Hall 2215 K-3
West Lafayette, IN 47907
o: 765-494-1934   m: 765-532-6587
[fIp8FQcCeZxx17HQ6oihGPfhG50MtHFy06Eaj4fQtlxGgE80sB8QS2OayeRlG2vsvy7L7xV6mTaVS8dU9VpRZvtvtdn3dy3Y8bVse+2gu7fu6F36D+XwexySxn2m51+tFWY4vdGSiXq9vbm7iIrZ4RcQbGre3t13G3PIvmhv4neEsNicV3dZH4iWQXalUxkoVekUfO21sZalL98YSG+edqqrabrexOA0wnl7sCPYr3gu3jfVgaZrWbDZrtZogCDjHYKK6owAAYHLEYAgAACAhAAAkBAAAQEIAAEgIAACAhAAAkBAAAAAJAQAgIQAAABICAEBCAAAAJAQAgIQAAABICAAACQEAAJAQAAASAgAAICEAACQEAABAQgAASAgAAICEAMClxP8PAJuuy+qBhrCOAAAAAElFTkSuQmCC]<https://www.purdue.edu/?utm_source=signature&utm_medium=email&utm_campaign=purdue>



From: Brian Gow ***@***.***>
Date: Monday, August 22, 2022 at 10:05 AM
To: MIT-LCP/mimic-code ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [MIT-LCP/mimic-code] About admission_location variable (Discussion #1365)
---- External Email: Use caution with attachments, links, or sharing data ----


@EstebanGarciaG<https://github.com/EstebanGarciaG> , I don't know the details behind 'TRANSFER FROM HOSPITAL' but can ask around to see if I can figure it out. It isn't clear what you need to know though. Can you let me know your criteria for including / excluding patients based on this and I'll try to get information around those specific criteria.
